pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -38,6 +39,7 @@ type kubeController struct {
 type Multicluster struct {
 	WatchedNamespace  string
 	DomainSuffix      string
+	caroot            string
```

Comment:
Suggest changing all caroot to caRoot to be consistent with the variable naming convention.